Makefile

```diff
@@ -11,10 +11,9 @@ VERSION := v3.6.0-beta.2
 KIND_VERSION ?= 0.11.0
 # note: k8s version pinned since KIND image availability lags k8s releases
 KUBERNETES_VERSION ?= 1.21.1
-KUSTOMIZE_VERSION ?= 3.8.8
+KUSTOMIZE_VERSION ?= v3.8.9
```

Comment:
why did we add back the `v` here? it is consistent with others without

Reply:
Having the "v" here makes it so KUSTOMIZE_VERSION is exactly the value of the tag, rather than having the "v" always prefix it everywhere it's used.

Reply:
can we add `v` to others? i think as long as it's consistent should be good

Reply:
I'll check

Reply:
Reverted "v" change. It can't be done for everything.